src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Loading/BitLoadingDemo.razor

```diff
@@ -5,6 +5,7 @@
             Description="loading component of the bit BlazorUI components" />
 
 <ComponentDemo ComponentName="Loading"
+               ComponentBaseParameters="null"
```

Comment:
this change is not related to this issue, please roll it back.

Reply:
Since we always update demo when we adding new parameters, I did these changes! and those totally related to BitLoading demo to show correct parameters :)

 am I wrong?

Reply:
yes, you are wrong